projects/dsp-ui/src/lib/viewer/views/list-view/list-view.component.ts

```diff
@@ -35,16 +58,28 @@ export class ListViewComponent implements OnChanges {
 
     @Input() displayViewSwitch?: boolean = true;
 
+    /**
+      * Set to true if multiple resources can be selected for comparison
+      */
+    @Input() withMultipleSelection?: boolean = false;
+
+    /**
+     * Click on checkbox will emit the resource info
+     *
+     * @param {EventEmitter<FilteredResouces>} resourcesSelected
+     */
+    @Output() multipleResourcesSelected?: EventEmitter<FilteredResouces> = new EventEmitter<FilteredResouces>();
+
     /**
      * Click on an item will emit the resource iri
      *
      * @param {EventEmitter<string>} resourceSelected
      */
-    @Output() resourceSelected: EventEmitter<string> = new EventEmitter<string>();
+    @Output() singleResourceSelected?: EventEmitter<string> = new EventEmitter<string>();
```

Comment:
Nice implementation, but I suggest to keep the `@Output() resourceSelected: EventEmitter<string> = new EventEmitter<string>();` and mark it in jsDocs as `@deprecated` (<https://jsdoc.app/tags-deprecated.html>). And additional the new line `@Output() singleResourceSelected?: EventEmitter<string> = new EventEmitter<string>();`. In this case we have the new implementation, but the apps will not break.

Reply:
Btw. does the `singleResourceSelected` not work when I have `withMultipleSelection = true`. So, I'm wondering how I can open a single resource when clicking on it. Do I have to use `multipleResourcesSelected` in this case? There's no combination of both? I assumed if I click on the resource (and didn't check the checkbox) that I would still be able to view the single resource.

Reply:
> Nice implementation, but I suggest to keep the `@Output() resourceSelected: EventEmitter<string> = new EventEmitter<string>();` and mark it in jsDocs as `@deprecated` (https://jsdoc.app/tags-deprecated.html). And additional the new line `@Output() singleResourceSelected?: EventEmitter<string> = new EventEmitter<string>();`. In this case we have the new implementation, but the apps will not break.

I will update the PR.

> Btw. does the `singleResourceSelected` not work when I have `withMultipleSelection = true`. So, I'm wondering how I can open a single resource when clicking on it. Do I have to use `multipleResourcesSelected` in this case? There's no combination of both? I assumed if I click on the resource (and didn't check the checkbox) that I would still be able to view the single resource.

Yes, you have to use `multipleResourcesSelected` and `withMultipleSelection = true` in case of multiple selection and `singleResourceSelected` and `withMultipleSelection = false` in case of single selection.
 I think it will be confusing for user if he/she clicks on resource to view it and select checkboxes to compare it.

Reply:
I think it's also confusing, when a user has to click on the checkbox to open only one resource. The checkboxes should be for multiple selection. As a user I expect to click on a resource to open it directly. Clicking on a checkbox to open it, is not the usual way to open something.

Reply:
in `singleResourceSelected`, checkbox is not displayed. User has to click on resource to view it as like we have now.

Reply:
In `multipleResourcesSelected`, I agree user needs to select one checkbox which will redirect it to the intermediate page and then display the resource(s). But that's the design we have in Figma now.

If we want user to directly select resource to view it by skipping intermedia page, we need to update the design in Figma.

Reply:
In `multipleResourcesSelected` we should have a combination of both. Clicking on a resource will open the single resource. But clicking on the checkboxes it will select 1-n resources and displays the intermediate page. This is a normal use case in UX design.

Reply:
I have updated the PR for this.